Innslag/Samling.php

```diff
@@ -464,16 +464,23 @@ private function _load( $pameldte=true, $countOnly=false ) {
             // har vi ikke tilgang på arrangementet, og dette må håndteres internt.
             // For å ikke kjøre alt for heavy objekter, prøver vi først uten listen
             // med kommuneID'er (november 2019) til første feil oppstår.
+            // 
+            // Fix 16.01: Forsøk å skippe innslag med kontakt-person context dersom arrangementet ikke finnes. Issue #315.
             if( $this->getContext()->getType() == 'kontaktperson' ) {
-                $innslag->getContext()->setMonstring(
-                    new Monstring(
-                        $innslag->getHomeId(),
-                        'kommune',
-                        $innslag->getSesong(),
-                        $innslag->getFylke()->getId(),
-                        null
-                    )
-                );
+                try {
+                    $innslag->getContext()->setMonstring(
+                        new Monstring(
+                            $innslag->getHomeId(),
+                            'kommune',
+                            $innslag->getSesong(),
+                            $innslag->getFylke()->getId(),
+                            null
+                        )
+                    );
+                } catch( Exception $e ) {
+                    // TODO: Error log - dette skjer kun dersom arrangementet ikke finnes lenger uten at innslaget er flyttet til en annen kommune. Dette BURDE vi få vite om.
```

Comment:
Dette skal i utgangspunktet skje automatisk nå, etter at vi endret på slette-algoritmen for litt siden :)